docs/sdks/server-sdks/python.md

```diff
@@ -50,9 +50,14 @@ else:
 * `<SUBJECT-KEY>` is the value that identifies each entity in your experiment, typically `user_id`;
 * `<FLAG-KEY>` is the key that you chose when creating a flag; you can find it on the [flag page](https://eppo.cloud/feature-flags). For the rest of this presentation, we’ll use `"test-checkout"`. To follow along, we recommend that you create a test flag in your account, and split users between `"fast_checkout"` and `"standard_checkout"`.
 
+Here's how this configuration looks in the [flag page](https://eppo.cloud/feature-flags):
```

Comment:
I figured it would be helpful to show a screenshot of the configuration. As a completely new person, this would have helped me. My only concern here is that the screenshot doesn't show a team, which seems to prevent me from creating the feature flag. However, it looks like the `test-checkout` feature flag was created in the Demo[DEV] workspace without a team? I'm going to ask Lukas about this today

Reply:
We can create an environment where the interface has Teams.
_Demo Company_ is ~~a mes…~~ busy but should have most of the latest features.

Reply:
> We can create an environment where the interface has Teams. _Demo Company_ is ~a mes…~ busy but should have most of the latest features.

@lukasgoetzweiss cleared this up for me earlier. Basically teams is a new feature, so previous flags are "grandfathered" in with no team specified. However, *new* flags require that a team be specified. We discussed a slight UI edit that will help prevent confusion, and I'm going to post about it in #team-feature-flags

Reply:
> We can create an environment where the interface has Teams. _Demo Company_ is ~a mes…~ busy but should have most of the latest features.


@lukasgoetzweiss cleared this up for me earlier. Basically teams is a new feature, so previous flags are "grandfathered" in with no team specified. However, *new* flags require that a team be specified. We discussed a slight UI edit that will help prevent confusion, and I'm going to post about it in #team-feature-flags